src/main/java/dan200/computercraft/shared/peripheral/monitor/ClientMonitor.java

```diff
@@ -164,5 +164,7 @@ public static void destroyAll()
                 iterator.remove();
             }
         }
+
+        MonitorTextureBufferShader.deleteUBO();
```

Comment:
I don't think we want this.

Both the UBO and shader are shared across multiple worlds, while monitors are not. It makes sense to delete the latter, but not the former.

Reply:
Ah, didn't think of that. Where do you suggest I put this?

Reply:
Do we need it at all? We never clean up the shader after all, and I assume the lifetimes of the two should be the same?

Reply:
It's kinda like not `free`ing memory that resides for the lifetime of the entire program anyway. Technically the OS will clean it up for us, it's just kinda ugly not to do it.

So no we don't need it, but it makes me feel better about myself. I'll happily remove it if you like.

Reply:
Please. I'm not even sure Minecraft has a shutdown handler.

It's not like we or MC free any of the other buffers we allocate afterall.